### ISSUE TITLE
ApplePayButtonStyle, ApplePayButtonType are unrecognized at build time

### ISSUE THREAD

**Issue by jjameson25** [flutter-stripe/flutter_stripe#57]
**Describe the bug**
Library is unable to resolve ApplePayButtonStyle and ApplePayButtonType

**To Reproduce**
1. Add the library to any project in pubspec.yaml
2. Attempt to build the project
3. Observe a failure with exceptions similar to 
```
../flutter/.pub-cache/hosted/pub.dartlang.org/flutter_stripe-1.0.1/lib/src/widgets/apple_pay_button.dart:166:20: Error: 'ApplePayButtonStyle' isn't a type.
    int mapButtonStyle(ApplePayButtonStyle style) {
```
**Expected behavior**
Package does not fail to resolve ApplePayButtonStyle + ApplePayButtonType

**Smartphone / tablet**
 - Device: iPhone XS Max or iPhone 12 simulator
 - OS: iOS 14
 - Package version: 1.0.1
 - Flutter version 2.2.0

**Comment by awazgyawali** [comment]
Happened to me too, quick solution would be to pull from master.
```
  flutter_stripe: 
    git:
      url: https://github.com/flutter-stripe/flutter_stripe
      path: stripe/
  ```

**Comment by jamesblasco** [comment]
Sorry about that, will push a new version with the last changes. 
We are adding testing so this does not happen again in the future

**Comment by jamesblasco** [comment]
This should be fixed in version 1.0.1+1

**Comment by remonh87** [comment]
version 1.0.1+3 broke this behavior again will fix it directly
